test/award_bonus/test_award_bonus.pkb

```diff
@@ -10,37 +10,36 @@ create or replace package body test_award_bonus as
     not_affected sys_refcursor;
     c_sales_amount constant number := 1000;
   begin
-    --arrange
+    --Arrange
     open expected for
       select (salary + c_sales_amount * gc_commision_pct) as new_salary
         from employees_test where employee_id = gc_test_employee;
 
     open not_affected for
       select * from employees_test where employee_id <> gc_test_employee;
 
```

Comment:
Erik: Changing that line-comment to initcap is going to be the killer enhancement for utPLSQL :)

Reply:
Perfect - Thank you @erikvanroon